src/components/App/App.component.jsx

```diff
@@ -34,6 +123,23 @@ function App() {
     <BrowserRouter>
       <AuthProvider>
         <Layout>
+          <Header>
```

Comment:
All of this can be extracted to a component that can be named `navbar`

Reply:
Done, looks cleaner now, thanks for the tip.